debian_worker_cloudbuild.yaml

```
@@ -0,0 +1,54 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Execute workflow debian_worker.wf.json to build a new Debian worker.
+# Tested with Debian 10 and Debian 11.
+#
+# To run the Cloud Build workflow, execute the following from the root
+# directory:
+# gcloud builds submit --config=debian_worker_cloudbuild.yaml .
+
+timeout: 4500s
+
+substitutions:
+  _IMAGE_PROJECT: 'compute-image-import-test'
+  _GCS_PATH: 'gs://compute-image-import-test'
```

These workers will be used not only in tests, so I would put them to our `compute-image-import` project.

```suggestion
  _IMAGE_PROJECT: 'compute-image-import'
  _GCS_PATH: 'gs://compute-image-import'
```